### ISSUE TITLE
[Feature Request] 1.1 Compatibility

### ISSUE THREAD

**Issue by enki1337** [Foltik/SpaghettiPack#1]
Modpack is incompatible with factorio 1.1

Looking forward to this great modpack seeing future revisions~! I'm sure there are plenty of upstream mods that we'll be waiting on for a revision, but I'm hoping we'll see a 1.1 update at some time.

Thank you for this great pack!

**Comment by foltik** [comment]
I'll definitely be releasing a 1.1 version with some new tweaks and fixes at some point once the rest of the dependencies get updated. Glad you like the pack!

**Comment by Alxandr** [comment]
Any updates on this? Should we still expect an update or have you dropped it (not a complaint if that is the case - just trying to manage expectations)? If we should not expect an update it would be nice to know, so that we can try to either fix it ourselves, or look at other packs :).

**Comment by foltik** [comment]
Because of the many changes in the underlying mods I used to build the pack, I've mainly shifted my focus towards building a brand new pack with custom built integrations instead of building on and updating this one.

While I won't be able to extensively tweak and balance things myself, I could release a 1.1 version with updated dependencies in the mean time if there was interest.

Let me know if that would be something you'd like to see.

**Comment by Alxandr** [comment]
Me and my friends were starting up today again after Christmas break. As a test (after taking backup of save and mods) we tried installing a newer pack with mostly the same mods, and it mostly just worked so we just ran with it. It was called something extreme multiplayer I believe (I can find the link tomorrow). The only issue we found was that infinite ores got deleted, but we didn't care about that. I would recommend just posting a note here and on the mod portal that this mod is no longer being maintained, and that one is mostly compatible *or* provide an update. Just so people who are playing your pack has a path forward that's not too painful. Would also recommend people to backup their saves at the same time.
